gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/LocalResourceManagerHelper.java

```diff
@@ -324,12 +322,9 @@ Response create(Project project) {
     }
   }
 
-  Response delete(String projectId) {
+  synchronized Response delete(String projectId) {
```

Comment:
This is a very coarse synchronization which could be done more efficient but I agree that for a test helper it is not worth the trouble.